draft-ietf-webpush-protocol.xml

```diff
@@ -1500,7 +1500,7 @@ HEADERS      [stream 82] +END_STREAM
     <section title="Acknowledgements">
       <t>
         Significant technical input to this document has been provided by Ben
-        Bangert, Peter Beverloo, Kit Cambridge, JR Conlin, Lucas Jenns, Matthew Kaufman,

```

Comment:
Sorry, I was unaware of that limitation. I've modified the PR to use the ASCII-friendly spelling of my name 😄